src/renderer/reader/components/Print.tsx

```diff
@@ -146,6 +146,25 @@ export const PrintContainer = ({ pdfPageRange, pdfThumbnailImageCacheArray }: {
         overscan: 3,
     });
 
+    function formatPrints(prints: number[]) {
```

Comment:
You cannot do that, [1-50] was an example with fifty pages "contigu". We need an algorithm that compress the page array of "contigu" pages.